diff --git a/protos.ethos b/protos.ethos
index 51754f3..5a1d28f 100644
--- a/protos.ethos
+++ b/protos.ethos
@@ -14,7 +14,7 @@ Library
            Bare.{ Extent String } ]
   Error.{ Extent Problem }
   Problem.[ Empty Multiple Unclosed.String Unexpected.String MissingHead MissingBody Budget Depth ]
-  ReaderBudget.{ Integer } ]
+  ReaderBudget.Integer ]
 [ Textualizable.[ textualize.[ String ] ]
   Canonicalizable.[] ]
 []
